content/open-source/oci/what-are-oci-artifacts.md

```diff
@@ -0,0 +1,49 @@
+
+OCI artifacts are sometimes misunderstood as a new specification or format but they are in fact just a way of using the OCI Image Specification to store something other than an image in a container registry. The Image Specification requires a particular format for the [Image Manifest](https://edu.chainguard.dev/open-source/oci/what-is-the-oci/#image-manifest), and the inclusion of the specified OCI `mediaTypes` values `application/vnd.oci.image.config.v1+json` and `application/vnd.oci.image.layer.v1.tar+gzip` in the `config` and `layers` fields. To store an artifact using the Image Specification, you need to change the `mediaTypes` values in the `config` and `layers` fields as in the example manifest below.  
+
+```json
+{
+  "schemaVersion": 2,
+  // no top-level mediaType to indicate OCI
+  "config": {
+    "mediaType": "application/vnd.yourcustomartifact+json",
+    "size": 233,
+    "digest": "sha245:..."
+  },
+  "layers": [{
+    "mediaType": "application/vnd.yourcustomartifact.tar.gzip",
+    "size": 680,
+    "digest": "sha245:..."
+  }]
+}
+```
+
+In the example manifest above, the  `config` field contains the custom `mediaType` value `application/vnd.yourcustomartifact+json` and the `layers` fields contains the custom `mediaType` value `application/vnd.yourcustomartifact.tar.gzip`.
+
+
+## OCI Artifacts and Software Supply Chain Security 
+For software supply chain security, OCI artifacts offer a useful way to store [SBOMs](​​https://edu.chainguard.dev/open-source/sbom/what-is-an-sbom/) and signatures inside a container registry. 
+
+An **SBOM**, or software bill of materials, is a formally structured list of libraries, modules, licensing, and version information that make up any given piece of software. When a vulnerability security advisory is issued, SBOMs enable software operators to quickly see whether their codebase contains any components associated with the vulnerability. A [**signature**](https://edu.chainguard.dev/open-source/sigstore/cosign/an-introduction-to-cosign/) is a way of verifying that you are the author of your software, and that it has not been tampered with by a third party. 
+
+The open source tool [Cosign](https://github.com/sigstore/cosign), part of the [Sigstore](https://www.sigstore.dev/) project, enables container engineers to sign their images and artifacts and store them as an artifact in an OCI registry. You can also use Cosign to store SBOMs as artifacts in an OCI registry. To learn more about storing signatures as artifacts, visit the [section on counter signing](
```

Comment:
I would use the broader "swe" vs "container engineer" here since Cosign works for any software artifact.

Do you think we need to double down on "store them as artifacts in an OCI registry" or can it be streamlined to "store them in an OCI registry"?

```suggestion
The open source tool [Cosign](https://github.com/sigstore/cosign), part of the [Sigstore](https://www.sigstore.dev/) project, enables software engineers to sign their images and other software artifacts and store them as artifacts in an OCI registry. You can also use Cosign to store SBOMs as artifacts in an OCI registry. To learn more about storing signatures as artifacts, visit the [section on counter signing](
```

Reply:
Fixed in local environment